app/models/concerns/foreman/thread_session.rb

```diff
@@ -116,6 +117,7 @@ def current=(organization)
           end
 
           Rails.logger.debug "Setting current organization thread-local variable to #{organization || 'none'}"
+          ::Logging.mdc['org_id'] = organization.try(:id)
```

Comment:
Adding `if SETTING[:organizations_enabled]` would be nice, as otherwise we see 2 fields on the JSON output like       `"loc_id":null`.